modules/@apostrophecms/image/ui/apos/components/AposMediaManagerDisplay.vue

```diff
@@ -95,6 +105,72 @@ export default {
         this.$emit('change', val);
       }
     }
+  },
+  mounted() {
+    // Get the acceptable file types, if set.
+    const imageGroup = apos.modules['@apostrophecms/attachment'].fileGroups
+      .find(group => group.name === 'images');
+
+    if (imageGroup && this.$refs['apos-upload-input']) {
+      const acceptTypes = imageGroup.extensions.map(type => `.${type}`)
+        .join(',');
+      console.info(acceptTypes);
+      this.$refs['apos-upload-input'].setAttribute('accept', acceptTypes);
+    }
+  },
+  methods: {
+    async uploadMedia (event) {
+      const file = event.target.files[0];
+
+      const emptyDoc = await apos.http.post(this.moduleOptions.action, {
+        body: {
+          _newInstance: true
+        }
+      });
+
+      // While the upload is working, set an uploading animation.
```

Comment:
Does this comment say what this is going to do, or is it a TODO?

Reply:
Actually I think you can remove this comment because the two `busy: true` flags should suffice.

Reply:
That is more of a TODO. I can make that clearer. There will be visual animations.